### ISSUE TITLE
Use ignore-styles to phase isomorphic loader

### ISSUE THREAD

**Issue by mmahalwy** [quran/quran.com-frontend#933]
https://github.com/bkonkle/ignore-styles

**Comment by hgeldiyev** [comment]
As-salamu aleykum. Could you please clarify what is it exactly you want with respect to this issue? I would like to help out, if possible. Also, where could I find conventions and information about what the SDLC cycle is? It would be helpful to know what commit to use as a base.

JZK

**Comment by mmahalwy** [comment]
@hgeldiyev sure! The problem we have is we rely on `webpack-isomorphic-tools` when server-side rendering for helping us with assets (images, styles) that webpack normally handles. I think it served us well but we probably should migrate to a prober solution since that felt like a hack at the time. Ignore styles might help

**Comment by mmahalwy** [comment]
Or better yet, we can do something like this:
https://medium.com/@mattvagni/server-side-rendering-with-css-modules-6b02f1238eb1

Or we can have the server-side handled by webpack, but we need to upgrade to webpack 3+

**Comment by hgeldiyev** [comment]
@mmahalwy 
Salams. I am sorry for my lack of understanding here. I am not as strong with JavaScript server-side development. However, I did look at the server-side-rendering article you posted and based on my understanding, it involves babel-plugin-css-modules-transform plugin to make CSS available to babel as variables. My question is - once I make the changes, what should be the observable outcome? I did generate the output with 'npm run build:server'; however, I am not sure what to look for.

Once again, thank you for the patience. Inshallah it will only get easier. :)
